alacritty.yml

```diff
@@ -253,6 +253,10 @@ mouse:
 
 selection:
   semantic_escape_chars: ",│`|:\"' ()[]{}<>"
+  # When set to true, selection string will be copy to primary clipborad.
+  # Default value is false, the selection string will be copy the selection
+  # clipborad.
+  save_to_clipboard: false
```

Comment:
To fall in line with the rest of the formatting in the `alacritty.yml`, an empty line should be added before this new block.

The default value also doesn't need to be documented when it's present in the default configuration file, I would propose rewording this to:
```yml
# When set to `true`, the selection string will be copied to the primary clipboard.
# Otherwise it will be copied to the selection clipboard.
```

This is just a proposal, if you have any better ideas please feel free to share them.